SuperBuild/External_ITK.cmake

```diff
@@ -48,7 +48,7 @@ if( ${ITK_WRAPPING} OR ${BUILD_SHARED_LIBS} )
   set( ITK_BUILD_SHARED_LIBS ON )
 else()
   set( ITK_BUILD_SHARED_LIBS OFF )
-  list( APPEND ep_itk_args"-DCMAKE_C_VISIBILITY_PRESET:BOOL=hidden" "-DCMAKE_CXX_VISIBILITY_PRESET:BOOL=hidden" )
+  list( APPEND ep_itk_args"-DCMAKE_C_VISIBILITY_PRESET:STRING=hidden" "-DCMAKE_CXX_VISIBILITY_PRESET:STRING=hidden" )
```

Comment:
:)  this probably has little actual impact, but it does change the ccmake GUI presentation of these values.